fabric-item-api-v1/src/main/java/net/legacyfabric/fabric/api/item/v1/Enchantable.java

```diff
@@ -14,13 +15,25 @@
  * limitations under the License.
  */
 
-package net.fabricmc.fabric.impl.client.indigo.renderer.accessor;
+package net.legacyfabric.fabric.api.item.v1;
 
-import net.minecraft.client.render.VertexConsumer;
-import net.minecraft.client.render.model.BakedModel;
-import net.minecraft.client.util.math.MatrixStack;
+import net.minecraft.item.Item;
 import net.minecraft.item.ItemStack;
 
-public interface AccessItemRenderer {
-	void fabric_renderBakedItemModel(BakedModel model, ItemStack stack, int light, int overlay, MatrixStack matrixStack, VertexConsumer buffer);
+public interface Enchantable {
```

Comment:
This name is misleading
Maybe `CustomEnchantabilityProvider`?

Reply:
i dont see how its misleading. can you elaborate?

Reply:
its because you dont need to implement this for the item to be enchantable.
@DeadlyMC suggest a name please